ldms/src/test/ldms-static-test-list.sh.in

```diff
@@ -3,15 +3,14 @@
 #check which plugins are enabled and store into an array.
 @ENABLE_MEMINFO_TRUE@ testlist[${#testlist[@]}]="meminfo"
 @ENABLE_APP_SAMPLER_TRUE@ testlist[${#testlist[@]}]="app_sampler"
-@ENABLE_PAPI_SAMPLER_TRUE@ testlist[${#testlist[@]}]="papi_sampler"
-@ENABLE_SYSPAPI_SAMPLER_TRUE@ testlist[${#testlist[@]}]="syspapi_sampler"
+@HAVE_LIBPAPI_TRUE@ testlist[${#testlist[@]}]="papi_sampler"
+@HAVE_LIBPFM_TRUE@ testlist[${#testlist[@]}]="syspapi_sampler"
```

Comment:
This should really be:

@HAVE_LIBPAPI_TRUE@@HAVE_LIBPFM_TRUE@

It depends on both.

Reply:
Okay I see my mistake, will make the necessary changes. Does the syspapi sampler also depend on both?

Thanks for letting me know.

Reply:
syspapi_sampler is the one that needs both, papi_sampler does not.

Reply:
Okay got it. Updated the script.